server/repository/src/migrations/v2_09_00/add_purchase_order_tables.rs

```diff
@@ -0,0 +1,103 @@
+            sql!(
+                connection,
+                r#"
+                CREATE TYPE purchase_order_status AS ENUM
+                    (
+                        'NEW',
+                        'CONFIRMED',
+                        'AUTHORISED',
+                        'FINALISED'
+                    );
+            "#
+            )?;
+
+            "purchase_order_status"
+        } else {
+            "TEXT"
+        };
+
+        sql!(
+            connection,
+            r#"
+                CREATE TABLE purchase_order (
+                    id TEXT NOT NULL PRIMARY KEY,
+                    store_id TEXT REFERENCES store(id),
+                    user_id TEXT,
+                    supplier_name_link_id TEXT REFERENCES name_link(id),
+                    -- corresponds to OG "serial_number"
+                    purchase_order_number INTEGER,
+                    status {purchase_order_status} NOT NULL,
+                    created_datetime {DATETIME} NOT NULL,
+                    confirmed_datetime {DATETIME},
+                    delivered_datetime {DATETIME},
+                    target_months {DOUBLE},
+                    comment TEXT,
+                    supplier_discount_percentage {DOUBLE},
+                    supplier_discount_amount {DOUBLE},
+                    donor_link_id TEXT REFERENCES name_link(id),
+                    reference TEXT,
+                    currency_id TEXT REFERENCES currency(id),
+                    foreign_exchange_rate {DOUBLE},
+                    shipping_method TEXT,
+                    sent_datetime {DATETIME},
+                    contract_signed_datetime {DATETIME},
+                    advance_paid_datetime {DATETIME},
+                    received_at_port_datetime {DATE},
+                    expected_delivery_datetime {DATE},
+                    supplier_agent TEXT,
+                    authorising_officer_1 TEXT,
+                    authorising_officer_2 TEXT,
+                    additional_instructions TEXT,
+                    heading_message TEXT,
+                    agent_commission {DOUBLE},
+                    document_charge {DOUBLE},
+                    communications_charge {DOUBLE},
+                    insurance_charge {DOUBLE},
+                    freight_charge {DOUBLE},
+                    freight_conditions TEXT
+                );
+            "#
+        )?;
+
+        sql!(
+            connection,
+            r#"
+                CREATE TABLE purchase_order_line (
+                    id TEXT NOT NULL PRIMARY KEY,
+                    item_code TEXT NOT NULL,
+                    item_name TEXT,
+                    number_of_packs {DOUBLE},
+                    pack_size {DOUBLE},
+                    -- corresponds to OG "original_quantity"
+                    requested_quantity {DOUBLE},
+                    -- corresponds to OG "adjusted_quantity"
+                    authorised_quantity {DOUBLE},
```

Comment:
These comments just to help whoever does the sync translations -- can probably delete after that's done.